screens/ViewProfileScreen.tsx

```diff
@@ -115,16 +117,24 @@ export default function ViewProfileScreen({
           </VStack>
 
           {profile.attachments.map((attachment) => (
-            <Entry
-              space={2}
-              px={2}
-              py={2}
-              key={attachment.blob.hash}
-              rounded={20}
+            <Pressable
+              onPress={() =>
+                navigation.navigate("ViewImage", { attachment: attachment })
+              }
             >
-              <BlobMedia handle={attachment.blob} />
-              <HStack justifyContent="center" alignItems="center"></HStack>
-            </Entry>
+              <VStack>
```

Comment:
Why is there a `VStack` here with only one child?

Reply:
There is no longer a `VStack` here with only one child.